Composer/packages/server/assets/projects/ToDoBotWithLuisSample/ComposerDialogs/Main/Main.dialog

```diff
@@ -1,18 +1,18 @@
 {
   "$type": "Microsoft.AdaptiveDialog",
   "$designer": {
-    "name": "Sample",
-    "description": "Test"
+    "$designer": {
+      "name": "ToDoBotWithLuisSample-17",
+      "id": "232009"
+    }
   },
-  "autoEndDialog": true,
+  "autoEndDialog": "true",
   "defaultResultProperty": "dialog.result",
-  "recognizer": "Main.lu",
   "triggers": [
     {
       "$type": "Microsoft.OnConversationUpdateActivity",
       "$designer": {
-        "id": "376720",
-        "name": "WelcomeUser"
```

Comment:
This is breaking an End to End test. Can you give this a name of `WelcomeUser`?

Reply:
Yes. Added.